### ISSUE TITLE
Unable to search vendor number in search bar starting with a character

### ISSUE THREAD

**Issue by janpanchalingam1** [unicef/etools-issues#598]
Browser: Chrome
Workspace: Iraq

Unable to search for vendor number "U21305" and return the correct results. Able to only search for "21305" and will return "U21305"

**Comment by acory** [comment]
@janpanchalingam1 @mpawlowski85  At the moment when you Add a new partner in pmp, the UI restricts the vendor number to digits. Should we allow letters also for vendor numbers?

![newpartner](https://user-images.githubusercontent.com/26309191/29204915-70cd5bcc-7e82-11e7-8db0-232146749f75.png)

**Comment by janpanchalingam1** [comment]
Not for the search for adding a vendor but for searching a vendor in the list view:
![image](https://user-images.githubusercontent.com/26144740/29215110-b7ed58d2-7e77-11e7-8ee7-bb81f2cdbf0f.png)

**Comment by acory** [comment]
@janpanchalingam1 Yes, I got that.This was just a question that came to me while working on this bug

**Comment by roxanavlad** [comment]
test result: pass
tested on dev environment on Google Chrome and Internet Explorer

**Comment by mpawlowski85** [comment]
passed in staging
